src/components/NftsPanel.vue

```diff
@@ -5,7 +5,6 @@
       :entities="nfts"
       :total-count="nftsCount"
       :limit="limit"
-      pagination-style="history"
```

Comment:
fix from previous work, there should not be history labels as the list is not sorted by the date yet